targets/targets.json

```diff
@@ -2058,7 +2058,8 @@
         "detect_code": ["0236"],
         "device_name": "LPC55S69JBD100",
         "release_versions": ["5"],
-        "program_cycle_s": 10
+        "program_cycle_s": 10,
+        "sectors": [[0,512]]
```

Comment:
Page erase is defined at 256 byte in
https://www.nxp.com/docs/en/user-guide/UM11126.pdf (page 3)
So 512 will work but we could optimise further.

Reply:
I agree with you. However, @offirko left the following comment on the internal ticket:

> Jimmy Brisson - please note that while LPC55S69 has 32KB sector size, it supports 512-bytes Page Erase command (subsectors).
> 
> When target's erase sector size is queried it returns Page Erase size of 512-bytes. (this is the value eventually used by FlashIAP block device,..etc)

> 
> https://github.com/ARMmbed/mbed-os/blob/f1e664be82d69da2517c0c2a481893bae11d7c3a/targets/TARGET_NXP/TARGET_MCUXpresso_MCUS/TARGET_LPC55S69/flash_api.c#L154
> 
>  
> 
> https://github.com/ARMmbed/mbed-os/blob/0e532ba2973a6fddd9ffa47365ba816e585f1e54/targets/TARGET_NXP/TARGET_MCUXpresso_MCUS/TARGET_LPC55S69/device/LPC55S69_cm33_core0_features.h#L229
> 
>  
> 
> Could we change the sector size defined at index.json to 512-bytes ?

I would prefer to leave it as 512 bytes to avoid introducing potential bugs. This is unless we also want to change the define for the flash page size. @evedon Please advise.

Reply:
Let's not do this change now.

Reply:
> Page erase is defined at 256 byte in
> https://www.nxp.com/docs/en/user-guide/UM11126.pdf (page 3)
> So 512 will work but we could optimise further.

This is a documentation error. Program and erase size is 512, thank you for finding this. We have raised an issue with the documentation team to correct this.